src/Telemetry/AzureDevOpsTimeline/AzureDevOpsTimelineOptions.cs

```diff
@@ -2,18 +2,20 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+using System.Collections.Generic;
+
 namespace Microsoft.DotNet.AzureDevOpsTimeline
 {
+    public class AzureDevOpsInstance
```

Comment:
Also, `record struct` my friend!  And then you don't have to do the complicated comparison later. ;-)

Reply:
This is used in options, so it may not play nicely with records.